R/getPrevalence.R

```diff
@@ -273,19 +260,12 @@ setMethod("getPrevalence", signature = c(x = "ANY"), function(
 #' @export
 setMethod("getPrevalence", signature = c(x = "SummarizedExperiment"),
     function(x, assay.type = assay_name, assay_name = "counts",
-            as.relative = as_relative, as_relative = FALSE, rank = NULL, ...){
-        # input check
-        if(!.is_a_bool(as.relative)){
-            stop("'as.relative' must be TRUE or FALSE.", call. = FALSE)
-        }
```

Comment:
We probably cannot just remove the functionality. One option is to move the functionality to hidden functionality (move the relative cabundance calculation to internal function).

Reply:
I am not sure I got you correctly because we have `.calc_rel_abund() `in internal functions for calculating relative abundance.

Reply:
Ahh, I have hidden the functionality in an internal function `.get_abundance()`